src/models/notion.ts

```diff
@@ -9,13 +9,31 @@ import {
   makeBlocks,
 } from "../utils";
 
+ 
+async function createNewbookHighlights(title: string, author: string, highlights: string[], blockType: BlockType, notionInstance: NotionAdapter) {
```

Comment:
@arkalim - hey there, this pr is still WIP. I cleaned up the code a bit from the code block I posted in the comment in this pr. I am not entirely sure why the BlockType is not accepted here as an argument. 

Here's the error I am getting - 
Argument of type 'typeof BlockType' is not assignable to parameter of type 'BlockType'.ts(2345)

Reply:
fixed it. The BlockType param is no necessary. Removed it from the function input argument. It's working now.